support-frontend/Brewfile

```diff
@@ -1,2 +1,6 @@
 tap "guardian/devtools"
 brew "guardian/devtools/dev-nginx"
+brew "nginx"
+brew "awscli"
+brew "sbt@1"
```

Comment:
minor: we could use the [`gu-scala` cask](https://github.com/guardian/homebrew-devtools/blob/master/Casks/gu-scala.rb) here instead